src/main/java/com/example/translate/service/TranslationServiceImpl.java

```diff
@@ -84,8 +79,8 @@ public List<Translations> getAll() {
         return translations;
     }
 
-    public Translations findBySourceText(String sourceText) {
-        return repository.findBySourceText(sourceText).orElseThrow(NotFoundException::new);
+    public List<Translations> findBySourceText(String sourceText) {
+        return repository.findBySourceText(sourceText);
```

Comment:
here first check if list is empty and than return NOT_FOUND after return your list of translations as response